Update/toki_tips_006.txt

```diff
@@ -191,7 +191,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>？？？</color>", NULL, "<color=#f5e6d3>???</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 32, "ps3/s20/32/vtt_hanada3056", 256, TRUE);
 	OutputLine(NULL, "「どちらが菩薩で孫悟空なのか、あとで思い知らせてやりますよ。",
-			NULL, "\"We'll find out later which of us is the god from the tale of the Monkey King.", Line_WaitForInput);
+			NULL, "\"We’ll find out soon which of us shall emerge a god.", Line_WaitForInput);
```

Comment:
I'm not sure if the literal TL feels quite right here, especially as the closing line to the arc. The Monkey King, through his craftiness and trickery, rose from being a simple rock to eventually becoming a Bodhisattva. So I we're meant to ponder which of these two people, through their machinations and sheer force of will, will soon become like a god. Edit suggested accordingly.